corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -407,19 +408,41 @@
             {# do not override this block, use registerurl template tag to populate #}
         {% endblock %}
         </div>
+        {% if request|toggle_enabled:"ANALYTICS_NEW" or USE_NEW_ANALYTICS %}
```

Comment:
Happy to see namespacing here; I've always felt a bit guilty that initial page data is a global namespace.